ERROR - 2023-11-05 06:15:19 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'dot_managementd...', 3306, '', 0)
#1 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\xampp\htdocs\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#3 C:\xampp\htdocs\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-11-05 06:15:19 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `name`, `image`, `created_at`, `description`
FROM `news`
WHERE `visible` = :visible:
ORDER BY `created_at` DESC', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4_capstone1.0\\public\\index.php')
ERROR - 2023-11-05 06:15:23 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'dot_managementd...', 3306, '', 0)
#1 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\xampp\htdocs\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#3 C:\xampp\htdocs\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-11-05 06:15:23 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `name`, `image`, `created_at`, `description`
FROM `news`
WHERE `visible` = :visible:
ORDER BY `created_at` DESC', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4_capstone1.0\\public\\index.php')
ERROR - 2023-11-05 06:15:27 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'dot_managementd...', 3306, '', 0)
#1 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\xampp\htdocs\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#8 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4_capstone1.0\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4_capstone1.0\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#3 C:\xampp\htdocs\ci4_capstone1.0\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ci4_capstone1.0\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\ci4_capstone1.0\app\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(934): App\Controllers\Home->index()
#7 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\ci4_capstone1.0\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci4_capstone1.0\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ci4_capstone1.0\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-11-05 06:15:27 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `name`, `image`, `created_at`, `description`
FROM `news`
WHERE `visible` = :visible:
ORDER BY `created_at` DESC', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Home.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\ci4_capstone1.0\\public\\index.php')
